wiki/BanchoBot/en.md

```diff
@@ -1,154 +1,144 @@
 ---
-needs_cleanup: true
+tags:
+  - bancho
+  - server bot
+  - commands
 ---
-<!-- TODO: the format of this article is just annoying to read. too many notes and useless wording -->
+
+<!--TODO:
+- add section that lists and explains all the game-related announcments -->
 
 # BanchoBot
 
+*Not to be confused with [Bancho (server)](/wiki/Bancho)*
+
 ![BanchoBot's user card](img/BanchoBot.jpg "BanchoBot's user card")
 
-BanchoBot is a bot (i.e. a set of commands with automated responses) that assists people in chat by displaying certain information and announcing game-related messages. It has been programmed by [Echo49](https://osu.ppy.sh/users/431) and is the host of the [Bancho IRC](/wiki/Internet_Relay_Chat) (Internet Relay Chat). BanchoBot also has its own [osu! profile](https://osu.ppy.sh/users/3) and [Twitter account](https://twitter.com/banchoboat)!
+**BanchoBot** (sometimes shortened to "Bancho") is an online chat bot for osu! that assists people in the in-game chat by announcing game-related messages (e.g., new first place ranks, total plays, retries, etc.), and responding to certain commands. It was programmed by [Echo49](https://osu.ppy.sh/users/431) and is the host of the [Bancho IRC](/wiki/Internet_Relay_Chat) (Internet Relay Chat). BanchoBot also has its own [osu! profile](https://osu.ppy.sh/users/3) and [Twitter account](https://twitter.com/banchoboat).
```

Comment:
```suggestion
**BanchoBot** (sometimes shortened to "Bancho") is an online chat bot for osu! that assists people in the in-game chat by announcing game-related messages (e.g., new first place ranks, total plays, retries, etc.), and responding to certain commands. It was programmed by [Echo](https://osu.ppy.sh/users/431) and is the provider of the [Bancho IRC](/wiki/Internet_Relay_Chat) (Internet Relay Chat). BanchoBot also has its own [osu! profile](https://osu.ppy.sh/users/3) and [Twitter account](https://twitter.com/banchoboat).
```
no need to feature Echo's previous (?) name, which might confuse readers

also I'm not sure if IRC is still hosted by Echo, since they are listed as inactive and alumni. so changing that part to "provider of the Bancho IRC" instead might be better